pkg/controller/kabaneroplatform/featured_stacks.go

```diff
@@ -168,8 +175,26 @@ func preProcessCurrentStacks(ctx context.Context, k *kabanerov1alpha2.Kabanero,
 		return err
 	}
 
+	// Only keep the FeaturedStack if the Kabanero pipelinesNamespace did not change, otherwise delete & recreate
+	pipelinesNamespace := pipelinesNamespace(k)
+	for _, deployedStack := range deployedStacks.Items {
+		if deployedStack.Spec.PipelinesNamespace != pipelinesNamespace {
+			err := cl.Delete(ctx, &deployedStack)
```

Comment:
Before we delete a stack with the incorrect pipeline namespace, we have to be sure that none of the versions have `desiredState` set to some value.  The `desiredState` is a clue that the stack is being manually managed.  I believe there is other code in here that checks the `desiredState` when attempting to sync up with the stack hub index yaml.